docs/approval-workflow.md

```diff
@@ -0,0 +1,51 @@
+---
+id: approval-workflow
+title: Orders Approval Workflow
+sidebar_label: Approval Workflow
+---
+In some specific use-cases, for example for products/services not supporting
+self-provisioning (e.g. "server management") an **order approval workflow**
+could be required.
+
+Each product/service can be specially configured to enable the *approval
+workflow*: in this case, each order (new purchases, change requests) needs to be
+approved by a *responsible person* (**Auditor**) before it becomes effective.
```

Comment:
```suggestion
approved by a privileged user (**Auditor**) before it becomes effective.
```

Reply:
non vuol dire quello che credi https://askinglot.com/how-would-you-describe-a-responsible-person